CHANGES.md

```diff
@@ -4,6 +4,19 @@
 
 No significant changes since the last release
 
+## 2.0.0
+
+| Released on | Released by   |
+|-------------|---------------|
+| 2022-03-23  | @dannyroberts |
+
+This is a major release because it changes behavior in a way that we regard as fixing an unintuitive behavior
+but could technically be breaking if the previous behavior was relied upon.
+
+- Assignment of a generator to a ListProperty property, which previously resulted in an error,
+  now results in the generator first being converted to a list. (https://github.com/dimagi/jsonobject/pull/200)
```

Comment:
@millerdev would you say this adequately describes the change in a way that's relevant to a user of this library? Is there anything inaccurate about it and/or anything you think should be added?

Reply:
This is not quite accurate. Assigning a non-list value to a `ListProperty` still raises `BadValueError` (same as before).